### PR TITLE
The Bad Guys: projects promo banner 

### PR DESCRIPTION
**Pull request by breville**
When DCDO flag `"thebadguys-promotion"` is set to `true` we get a temporary banner at https://studio.code.org/projects/public.

With flag:

Screen Shot 2022-04-06 at 12 02 18 PM

Without flag:

Screen Shot 2022-04-06 at 12 02 44 PM

### CODE REVIEW

File: apps/i18n/common/en_us.json [hunk 1]
```diff
@@ -1484,7 +1484,9 @@
   "projectName": "Project Name",
   "projectPromoButton": "View Project Ideas",
   "projectPromoDescription": "Take a look at our Project Ideas page for starter projects in Sprite Lab, Game Lab, App Lab, and Web Lab. These include project descriptions, tips, and demo projects you can remix to make your own!",
+  "projectPromoDescriptionThebadguys": "Make a story or animation starring Ms. Tarantula, Mr. Wolf, Mr. Shark, and the rest of the gang from the new movie The Bad Guys! Recreate your favorite parts of the film, or make something totally unique — it's all up to you!",
```
Comment (epeach) [inline apps/i18n/common/en_us.json:1487]: Does "The Bad Guys" need to be italicized or parenthesized or anything?

Comment (epeach) [reply apps/i18n/common/en_us.json:1487]: Also Nit: with the key `projectPromoDescriptionThebadguys`, I would recommend capitalizing the `b` and `g` in `badguys` to keep it consistent with our naming practices. (This seems to be a pattern throughout this PR)

Comment (breville) [reply apps/i18n/common/en_us.json:1487]: > Does "The Bad Guys" need to be italicized or parenthesized or anything?

Ideally I think so, but I don't believe we have an easy way to support this due to the loc pipeline.  I'll do a follow up if I hear that it becomes crucial.

> Also Nit: with the key `projectPromoDescriptionThebadguys`, I would recommend capitalizing the `b` and `g` in `badguys` to keep it consistent with our naming practices. (This seems to be a pattern throughout this PR)

I thought about this, and it does read a bit odd, but we decided to use `thebadguys` as our ID throughout the source code, so I'm treating it as a single keyword.

Comment (epeach) [reply apps/i18n/common/en_us.json:1487]: I see, re: `thebadguys`. A little hard to read, but I can see how it makes sense to keep it as a single id.

**Review by epeach — APPROVED** [review]

**Review by kelbyhawn — APPROVED** [review]

**Comment by kelbyhawn** [comment]
[The Bad Guys](https://trello.com/c/LFUR4e7L/1-the-bad-guys)

**Pull request merged**